config/jest/log-on-fail-reporter.js

```diff
@@ -3,7 +3,7 @@
  * From: https://gist.github.com/GeeWee/71db0d9911b4a087e4b2486386168b05
  */
 const chalk = require("chalk");
-const {getConsoleOutput} = require("jest-util");
+const {getConsoleOutput} = require("@jest/console");
```

Comment:
Changed this line because Jest v25 moved this function to the `console` package.